project/Dependencies.scala

```diff
@@ -2,11 +2,26 @@ import sbt._
 
 object Dependencies {
 
-  val catsEffect = "org.typelevel" %% "cats-effect" % "2.0.0"
-  val kindProjector = "org.typelevel" %% "kind-projector" % "0.10.3"
-  val pureConfig = "com.github.pureconfig" %% "pureconfig" % "0.12.0"
-  val scalaTest = "org.scalatest" %% "scalatest" % "3.0.8" % Test
-  val zio = "dev.zio" %% "zio" % "1.0.0-RC13"
-  val zioInteropCats = "dev.zio" %% "zio-interop-cats" % "2.0.0.0-RC4"
+  object Versions {
```

Comment:
I don't like this. It seems unnecessary to me because you are duplicating the name of the library 3 times in this file. I was even against having the `Dependencies` file because 99% of the dependencies are defined only once in the whole project so the extraction is unenecessary but in the end I said ok let's have it refactored into a file but extracting also versions seems just crazy to me. Please don't do this. And please keep the dependencies sorted. Also, in master ScalaTest is already refactored into "test" dependencies.

Reply:
I think this is the most valuable part of pushing "dependencies" away into a common place -- very often you want to use the same version for what is logically one library, but technically is multiple jars (like this project, or cats/cats-mtl, and so on)

doing it this style makes you declare that version only once